DependencyInjection/Configuration/Configuration.php

```diff
@@ -105,6 +105,10 @@ private function addClientsSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                                 ->beforeNormalization()
                                     ->always()->then(function($v) {
+                                        if (!is_array($v)) {
+                                            return $v;
+                                        }
```

Comment:
No, because null is not a string:
`->ifString()->then(function($v) { return (array) $v; })`